test/Microsoft.Azure.Devices.Edge.Test/helpers/ManualProvisioningFixturePreview.cs

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Test.Helpers
+{
+    using System;
+    using System.Linq;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.Test.Common;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using NUnit.Framework;
+
+    // NUnit's [Timeout] attribute isn't supported in .NET Standard
+    // and even if it were, it doesn't run the teardown method when
+    // a test times out. We need teardown to run, to remove the
+    // device registration from IoT Hub and stop the daemon. So
+    // we have our own timeout mechanism.
+    public class ManualProvisioningFixturePreview : BaseFixture
+    {
+        protected readonly IotHub iotHub;
+        protected IEdgeDaemon daemon;
+
+        public ManualProvisioningFixturePreview()
```

Comment:
The only difference I see between this and `ManualProvisioningFixture` is that you pass different arguments into the `IotHub` constructor. And I don't see any differences between `SasManualProvisioningFixturePreview` and `SasManualProvisioningFixture`.

Seems like you could save a lot of duplicate code by adding two parameters to the fixtures' constructors, with the non-preview values as defaults. Then in your test class, add a constructor that calls `base(previewConnectionString, previewEventHubEndpoint)`.